src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
In my own plugin, I have seen a possible problem with the paths provided by Sonar (written in Unix format) and ESLint paths in the results file that are depending of the OS. I collected both Windows and Unix paths for the file I am analyzing depending of the OS.  I had to fix the paths into the Sensor by using the java.io.File provided by Sonar.  Don't you have the same issue ?

https://github.com/sleroy/SonarEsLintPlugin/blob/master/src/main/java/io/github/sleroy/sonar/EsLintSensor.java#L82